packages/flutter_tools/templates/plugin_ffi/src.tmpl/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,23 @@
+# We use the CMake version connected to the version of the Android Gradle Plugin.
+#
+# CMake 3.10.2.
+# https://developer.android.com/studio/projects/install-ndk#vanilla_cmake
+#
+# Changing the CMake version requires all users of this plugin to
+# download that CMake version.
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+
+project({{projectName}}_library VERSION 0.0.1 LANGUAGES C)
+
+add_library(
+    {{projectName}}
+    SHARED
```

Comment:
It would be nice to format this CMake file more like the Linux and Windows ones for consistency.